reconcile/change_owners/changes.py

```diff
@@ -244,6 +244,8 @@ def _create_bundle_file_change(
         # - the SHA of the file: the SHA can change even when no diffs are detected.
         #   this can be happen in case something undetectable has changed, e.g. comments
         #   in a YAML file are invisible to YAML parsers
+        # - item reodering in lists: we ignore reordering of elements in lists right now
```

Comment:
nit: typo on the first `reorrrrrrdering`

Reply:
it is time for a vscode spell-checking plugin